src/Microsoft.Dnx.Runtime.Sources/Impl/EnvironmentNames.cs

```diff
@@ -16,12 +16,9 @@ internal static class EnvironmentNames
         public const string AppBase = "DNX_APPBASE";
```

Comment:
https://github.com/aspnet/dnx/blob/ba748dbe1abbcd243b775a27eb97e085aeb58b77/src/Microsoft.Dnx.Tooling/Publish/PublishRoot.cs#L174

This is to pass to the script template to fill `{0}` here:

```
export SET {0}=""$DIR/{1}""
exec ""{2}{3}"" --appbase ""${0}"" Microsoft.Dnx.ApplicationHost --configuration {4} {5} ""$@""";
```